prometheus_client/bridge/newrelic.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import absolute_import
+
+import newrelic.agent
+
+from .. import core
+
+
+@newrelic.agent.data_source_factory(name='Prometheus')
+class PrometheusDataSource(object):
+    def __init__(self, settings, environ):
+        self.registry = settings.get('registry', core.REGISTRY)
+        self.prefix = settings.get('prefix', '')
+
+    def __call__(self):
+        prefixstr = ''
+        if self.prefix:
+            prefixstr = self.prefix + '.'
+
+        for metric in self.registry.collect():
+            for name, labels, value in metric.samples:
```

Comment:
In the cases of a Summary and Histogram, would it make sense to return a dict for value with the `_count`/`_sum` as Count/Value for those two samples?